dotCMS/src/main/java/com/dotcms/rest/api/v2/contenttype/FieldResource.java

```diff
@@ -270,15 +263,15 @@ public Response updateContentTypeFieldByVar(@PathParam("typeId") final String ty
 
             } else {
 
-                Field currentField = fapi.byContentTypeIdAndVar(typeId, fieldVar);
+                final Field currentField = fieldAPI.byContentTypeIdAndVar(typeId, fieldVar);
 
                 if (!currentField.id().equals(field.id())) {
 
-                    response = ExceptionMapperUtil.createResponse(null, "Field var '"+ fieldVar +"' does not match a field with id '"+ field.id() +"'");
+                    throw new DotDataValidationException("Field var '"+ fieldVar +"' does not match a field with id '"+ field.id() +"'");
```

Comment:
let's fix this message and make it look like the other one: "Field id '"+ fieldId +"' does not match a field with id '"+ field.id() +"'"